warehouse/templates/manage/settings.html

```diff
@@ -21,14 +21,13 @@
   <h2>Project Settings</h2>
 
   <div class="callout-block">
-    <h3>Project Links</h3>
+    <h3>Project Description and Sidebar</h3>
     <p>
-      To set the Project Links in the '{{ project.name }}' sidebar for
-      your next release, use
-      the <a href="https://packaging.python.org/tutorials/distributing-packages/#url"><tt>url</tt></a>
-      and <a href="https://packaging.python.org/tutorials/distributing-packages/#project-urls"><tt>project_urls</tt></a>
-      fields in your <tt>setup.py</tt> file. Updating these fields
-      will not change the Project Links for past
+      To set the '{{ project.name }}' description, author, links,
+      classifiers, and other details for your next release, use
+      the <a href="https://packaging.python.org/tutorials/distributing-packages/#setup-args"><tt>setup()</tt>
+      arguments in your <tt>setup.py</tt> file</a>. Updating these
+      fields will not change the Project Links for past
```

Comment:
Not just "Project Links" -- this won't change any of the metadata for past releases.

Reply:
Thanks. Fixed.